installer/Migration/V5_5_0/Migration.php

```diff
@@ -128,15 +128,15 @@ public function up(): void
                 'ohrm_claim_event',
                 ['id'],
                 'claimEventId',
-                ['onDelete' => 'RESTRICT']
+                ['onDelete' => 'CASCADE']
```

Comment:
@ManulMax check 4.x behavior
seems `set null` is 4.x behavior.

Reply:
Checked with BA and said need to delete all claim request related to employee if we delete that employee.